### ISSUE TITLE
Tilt compensation change from RX to pole tip is hazardous to data integrity

### ISSUE THREAD

**Issue by gdt** [sparkfun/SparkFun_RTK_Everywhere_Firmware#499]
(I don't have a torch, but I know someone who is thinking about it.)

Currently, with the non-torch units without tilt compensation, the workflow is:
  - unit reports position of APC
  - user levels the pole
  - data collector is configured with APC-tip offset (= APC-ARP offset plus ARP-tip offset!!)

and this is ok, except you don't have tilt compensation.

With the torch, as I understand it from reading the docs, when you first get an RTK fix, the unit is outputting the APC position.  Then, after tilt compensation, it's outputting pole tip.  But if it pops out of tilt mode, perhaps due to momentary loss of RTK, it can go back to APC.    This means one's data is suspect, basically.

I suggest (assuming I'm not confused):
  - user manual explains that "pole height" is APC-ARP plus ARP-tip, and SF or somebody experiments with and documents the APC-ARP average value
  - user configures APC-tip as happens now
  - when not in tilt compensation mode, the APC-tip offset is still subtracted, so one gets pole tip assuming pole is vertical
  - if tilt mode is lost, then output just stops, until either regained, or user explicitly turns off tilt mode.

The point is that incorrect data should not be recorded.   Already data collectors have to record the error in the M field of POINTZM, or refrain from recording if high, but tilt/not is an additional dimension.

**Comment by nseidle** [comment]
Yes, you are correct. If tilt compensation mode is exited, the devices reported location changes from the pole tip to the APC. This can, and is usually, significant. Per your recommendation, I've added some additional blocks of into the [tilt menu docs](https://docs.sparkfun.com/SparkFun_RTK_Everywhere_Firmware/menu_tilt/). Tilt compensation is a wonderful, magical way to shoot yourself in the foot. If there's other docs you think we can add to make this more obvious, please let me know.
